aten/src/ATen/native/native_functions.yaml

```diff
@@ -4382,9 +4382,6 @@
 # softmax allows positional dtype, unlike most operators, because kwonly is BC-breaking when loading jit models.
 - func: softmax.int(Tensor self, int dim, ScalarType? dtype=None) -> Tensor
   variants: function, method
-  dispatch:
-    CompositeImplicitAutograd: softmax
-    NestedTensorCPU, NestedTensorCUDA: softmax
```

Comment:
If this gets removed, do we still go into the nested-specific softmax kernel?

hmm I also see that it was previously just calling `softmax` lol

Reply:
So this top-level softmax.int is essentially a thin wrapper that calls _softmax and that has the mutliple dispatches. I removed this top level dispatch and then added just the _softmax kernel. The problem was that NestedTensorCPu wasn't behaving well with composite implicit kernels but a change I just made should fix this

Reply:
Ah, makes sense!